networkit/cpp/centrality/GroupDegree.cpp

```diff
@@ -58,10 +58,7 @@ void GroupDegree::run() {
 		updateGroup();
 	}
 
-	std::vector<node> neighbors = G.neighbors(group.back());
-#pragma omp parallel for
-	for (omp_index i = 0; i < neighbors.size(); ++i) {
-		node u = neighbors[i];
+	for (node u : G.neighborRange(group.back())) {
```

Comment:
Here, we get rid of the parallelism -- which is probably not an issue as there are not so many neighbors.